public/controllers/management/components/management/groups/utils/columns-files.js

```diff
@@ -10,6 +11,50 @@ export default class GroupsFilesColumns {
     const { itemDetail } = this.tableProps.state;
     this.groupsHandler = GroupsHandler;
 
+    this.showFile = async(item) => {
+      let result = await this.groupsHandler.getFileContent(
+        `/groups/${itemDetail.name}/files/${item.filename}/xml`
+      );
+
+      if (Object.keys(result).length == 0) {
+        result = '';
+      }
+
+      const data =  typeof result === 'object'
+          ? JSON.stringify(result, null, 2)
+          : result.toString()
+
+      const file = {
+        name: item.filename,
+        content: data,
+        isEditable: false,
+        groupName: itemDetail.name
+      };
+
+      this.tableProps.updateFileContent(file);
+    }
+
+    this.editFile = async (item) => {
+
+      let result = await this.groupsHandler.getFileContent(
+        `/groups/${itemDetail.name}/files/${item.filename}/xml`
+      );
+      if (Object.keys(result).length == 0) {
+        result = '';
+      }
+
+      const data = this.autoFormat(result);
+
+      const file = {
+        name: 'agent.conf',
+        content: data,
+        isEditable: true,
+        groupName: itemDetail.name,
+      };
+
+      this.tableProps.updateFileContent(file);
+    }
```

Comment:
**nitpick**: These 2 methods seem to be similar, maybe we could refactor them to share the logic.